lib/Devel/Cover/Report/Coveralls.pm

```diff
@@ -35,6 +35,11 @@ sub get_source {
 
     $file =~ s!^blib/!!;
 
+    # Prepend sub-directory name to file name
```

Comment:
can we have a better name ?
COVERALLS_FLAG_NAME
is not suitable for this senario... more like path name, sub directory name something

Reply:
How about `$ENV{CHANGED_DIR}` that we set over here https://github.com/regentmarkets/github-actions/blob/master/ci-setup/action.yml#L45?

Reply:
Updated to use `CHANGED_DIR` instead of `COVERALLS_FLAG_NAME`. Please help to review again, thanks!